ansible_collections/arista/avd/plugins/action/eos_cli_config_gen.py

```
@@ -25,12 +25,12 @@
 )
 
 if TYPE_CHECKING:
-    from pyavd import get_device_config, get_device_doc
+    from pyavd import ConfigRenderConfiguration, DocRenderConfiguration, get_device_config, get_device_doc
     from pyavd._utils import strip_empties_from_dict, template
     from pyavd.j2filters import add_md_toc
 
 try:
-    from pyavd import get_device_config, get_device_doc
+    from pyavd import ConfigRenderConfiguration, DocRenderConfiguration, get_device_config, get_device_doc
```

Flake8 (7.3.0)

[error] redefinition of unused 'ConfigRenderConfiguration' from line 28

(F811)